requirements_test.txt

```diff
@@ -7,14 +7,14 @@
 
 -c homeassistant/package_constraints.txt
 -r requirements_test_pre_commit.txt
-astroid==3.2.3
+astroid==3.2.4
```

Comment:
changelog for astroid is missing from opening text

Reply:
Added the link